subsys/usb/class/audio/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_include_directories(
+  # USB headers
+  ${ZEPHYR_BASE}/include/usb
+  ${ZEPHYR_BASE}/subsys/usb
```

Comment:
Do you need this actually? See [subsys/usb/CMakeLists.txt](https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/usb/CMakeLists.txt#L4).

Reply:
I used hid and netusb as a reference. I will check if this is still needed.

Reply:
Neither of those seems to be needing this entry, as it is already added in the main USB subsys CMake list.
`${ZEPHYR_BASE}/include/usb` seems also to be unnecessary, since you're including this stuff via `#include <usb/*.h>`.